lib/heartcheck/activerecord/version.rb

```diff
@@ -2,6 +2,6 @@
 module Heartcheck
   # gem version
   module Activerecord
-    VERSION = '0.0.3'
+    VERSION = '0.1.0'.freeze
```

Comment:
What about adding the change in CHANGELOG in this MR also?

Reply:
I haven't found a CHANGELOG file in this project, so I'm not sure about how new version updates are being documented... Anyway, I'm going to add a CHANGELOG file to the project using the [Keep a Changelog](https://keepachangelog.com/pt-BR/1.0.0/) format. Can you take a look?